Project.toml

```diff
@@ -57,6 +57,7 @@ ForwardDiff = "^0.10"
 JSON3 = "1"
 LinearAlgebra = "1"
 LogExpFunctions = "^0.3.24"
+Metaheuristics = "2, 3"
```

Comment:
Wait, why did we add Metaheuristics as a dependency?

Reply:
It is just as compat, it only applies when the extension is loaded